custom_components/oncharger/sensor.py

```diff
@@ -141,12 +254,42 @@ async def async_setup_entry(
             if (description := ENTITY_DESCRIPTIONS.get(ent))
         ]
     )
+    if dev_type == SINGLE_PHASE:
```

Comment:
We need to make it backward compatible for people who already configured, so I'd suggest to check `THREE_PHASE` and move this to else